ballerina/Module.md

```diff
@@ -1,17 +1,173 @@
+     "refresh_token": "<Refresh Token>",
+     "access_token": "<Access Token>",
+     "expires_in": 1800
+   }
+   ```
+
+5. Store the access token and refresh token securely for use in your application.
+
 
 ## Quickstart
 
-[//]: # (TODO: Add a quickstart guide to demonstrate a basic functionality of the module, including sample code snippets.)
+To use the `HubSpot CRM Object Tickets` connector in your Ballerina application, update the `.bal` file as follows:
 
-## Examples
+### Step 1: Import the module
 
-The `HubSpot CRM Object Tickets` connector provides practical examples illustrating usage in various scenarios. Explore these [examples](https://github.com/module-ballerinax-hubspot.crm.object.tickets/tree/main/examples/), covering the following use cases:
+Import the `hubspot.crm.obj.tickets` module and `oauth2` module.
+
+```ballerina
+import ballerinax/hubspot.crm.obj.tickets as hstickets;
+import ballerina/oauth2;
+```
+
+### Step 2: Instantiate a new connector
+
+1. Create a `Config.toml` file inside the Ballerina package and add the following configurations with the values retrieved in the earlier steps as follows:
+
+   ```toml
+    clientId = <Client Id>
+    clientSecret = <Client Secret>
+    refreshToken = <Refresh Token>
+   ```
+
+2. Instantiate a `OAuth2RefreshTokenGrantConfig` with the obtained credentials and initialize the connector with it.
+
+    ```ballerina
+   configurable string clientId = ?;
+   configurable string clientSecret = ?;
+   configurable string refreshToken = ?;
+
+   OAuth2RefreshTokenGrantConfig auth = {
+      clientId,
+      clientSecret,
+      refreshToken,
+      credentialBearer: oauth2:POST_BODY_BEARER 
+   };
+
+   final Client HubSpotClient = check new ({auth});
+   ```
+
+Now, utilize the available connector operations. A sample use case is shown below.
+
+#### Create a New Ticket
+
+```ballerina
+public function main() returns error? {
+    hstickets:SimplePublicObjectInputForCreate payload = {
+        properties: {
+            "hs_pipeline": "0",
+            "hs_pipeline_stage": "1",
+            "hs_ticket_priority": "HIGH",
+            "subject": "New troubleshoot report"
+        }
+    };
+    hstickets:SimplePublicObject response = check HubSpotClient->/crm/v3/objects/tickets.post(payload);
+    io:println(response);
+}
+```
+
+## Examples
 
-[//]: # (TODO: Add examples)
+The `HubSpot CRM Object Tickets` connector provides practical examples illustrating usage in various scenarios. Explore these [examples](https://github.com/ballerina-platform/module-ballerinax-hubspot.crm.obj.tickets/tree/main/examples), covering the following use cases:
+   1. [Ticket Management System](https://github.com/ballerina-platform/module-ballerinax-hubspot.crm.object.tickets/tree/main/examples/ticket-management-system)
+   2. [Weekly Tickets Report Generation](https://github.com/ballerina-platform/module-ballerinax-hubspot.crm.object.tickets/tree/main/examples/weekly-ticket-reports)
```

Comment:
Add a brief description for each example here. Check the [reference connector](https://github.com/ballerina-platform/module-ballerinax-twitter/tree/main/examples#examples)